code/modules/mob/living/say.dm

```diff
@@ -88,9 +90,12 @@ proc/get_radio_key_from_channel(var/channel)
 	if(GREY in mutations)
 		message = "<span class='grey'>[message]</span>"
 
-	else if(COMIC in mutations)
+	if(COMIC in mutations)

```

Comment:
why did you remove this else condition
I understand you had trouble with getting IsVocal to work, however, what does this have to do with it? An else here should only be part of the if statement above, "if grey in mutations, else if comic in mutations" followed by a regardless "if not vocal"